python/examples/yolo/models/classes.txt

```diff
@@ -0,0 +1,80 @@
+person
```

Comment:
there is a coco_name_list under common folder, should be same as this one

Reply:
Changed to use coco_name_list under common folder.